module/batch-action/src/Resources/config/messenger.yaml

```diff
@@ -4,3 +4,4 @@ framework:
             batch-action: "batch-action://"
         routing:
             Ergonode\BatchAction\Domain\Command\ProcessBatchActionEntryCommand: 'batch-action'
+            Ergonode\BatchAction\Domain\Command\ProcessBatchActionCommand: 'batch-action'
```

Comment:
transport name has changed with underscore: `batch_action`. Probably worth refreshing the feature branch before proceeding